docs/development/geo.md

```diff
@@ -23,53 +23,83 @@ title: "Spatial filters"
+
+Spatial dimensions are sting columns that contain coordinates separated by a comma.
+In the ingestion spec, you configure spatial dimensions in the `dimensionsSpec` object of the `dataSchema` component.
+For information on how to use the ingestion spec to configure ingestion, see [Ingestion spec reference](../ingestion/ingestion-spec.md).
+For general information on loading data in Druid, see [Ingestion](../ingestion/index.md).
+
+You can provide spatial dimensions in any of the [data formats](../ingestion/data-formats.md) supported by Druid.
+The following example shows spatial dimensions in JSON format:
 
 ```json
 {
-	"type": "hadoop",
-	"dataSchema": {
-		"dataSource": "DatasourceName",
-		"parser": {
-			"type": "string",
-			"parseSpec": {
-				"format": "json",
-				"timestampSpec": {
-					"column": "timestamp",
-					"format": "auto"
-				},
-				"dimensionsSpec": {
-					"dimensions": [],
-					"spatialDimensions": [{
-						"dimName": "coordinates",
-						"dims": ["lat", "long"]
-					}]
-				}
-			}
-		}
-	}
+  "type": "hadoop",
+  "dataSchema": {
+    "dataSource": "DatasourceName",
+    "parser": {
+      "type": "string",
+      "parseSpec": {
+        "format": "json",
+        "timestampSpec": {
+          "column": "timestamp",
+          "format": "auto"
+        },
+        "dimensionsSpec": {
+          "dimensions": [
+            {
+              "type": "double",
+              "name": "x"
+            },
+            {
+              "type": "double",
+              "name": "y"
+            }
+          ],
+          "spatialDimensions": [
+            {
+              "dimName": "coordinates",
+              "dims": [
+                "x",
+                "y"
+              ]
+            }
+          ]
+        }
+      }
+    }
+  }
 }
 ```
 
+The `spatialDimensions` array has the following elements:
+
+|Property|Description|Required|
+|--------|-----------|--------|
+|`dimName`|The name of a spatial dimension. You can construct a spatial dimenion from other dimensions or it may already exist as part of an event. If a spatial dimension already exists, it must be an array of coordinate values.|yes|
+|`dims`|The list of dimension names that comprise the spatial dimension.|no|
+
 ## Spatial filters
 
-|property|description|required?|
-|--------|-----------|---------|
-|dimName|The name of the spatial dimension. A spatial dimension may be constructed from multiple other dimensions or it may already exist as part of an event. If a spatial dimension already exists, it must be an array of coordinate values.|yes|
-|dims|A list of dimension names that comprise a spatial dimension.|no|
+A filter is a JSON object indicating which rows of data should be included in the computation for a query.
+You can filter on spatial structures, such as rectangles and polygons, using the spatial filter.
 
-The grammar for a spatial filter is as follows:
+The spatial filter has the following structure:
```

Comment:
This is a specific example and not a generic structure. We could add something like the following:

```
"filter": {
  "type": "spatial",
  "dimension": <name_of_spatial_dimension>,
  "bound": <bound_type>
}
```

That way we can have the generic grammar, then introduce a specific example.